js/projectsConstructor.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+function Project (name, url, github, img, codebases) {
+  this.name = name;
+  this.url = url;
+  this.github = github;
+  this.img = img;
+  this.codebases = codebases;
+  this.renderProject = function () {
```

Comment:
You should be putting this method on the `prototype` not within your constructor.